backend/src/test/java/de/uslu/Travel/TravelServiceTest.java

```diff
@@ -0,0 +1,47 @@
+package de.uslu.Travel;
+
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+
+import java.util.Collection;
+import java.util.List;
+
+
+class TravelServiceTest {
+
+    @Test
+    void shouldReturnAllTravels() {
+
+        Travel travel1 = new Travel();
+        Travel travel2 = new Travel();
+
+        TravelRepo mockRepo = Mockito.mock(TravelRepo.class);
+        Mockito.when(mockRepo.findAllByCity("hamburg")).thenReturn(List.of(travel1,travel2));
+
+        TravelService travelService = new TravelService(mockRepo);
+        Collection<Travel> actual = travelService.listTravels("hamburg");
+
+        Assertions.assertThat(actual.size()).isEqualTo(2);
+    }
+
+    @Test
+    void shouldReturnCreatedTravel() {
+
+        Travel travel = new Travel();
+        travel.setCity("Hamburg");
+        travel.setNotes("kalt");
+
+        Travel travelSaved = new Travel();
+        travelSaved.setCity("Hamburg");
+        travelSaved.setNotes("kalt");
```

Comment:
```suggestion
        Travel travel = new Travel();
        travel.setCity("Hamburg");
        travel.setNotes("kalt");
```

Both the same, since you don't generate an ID